server/helpers/Token.js

```diff
@@ -76,5 +76,38 @@ class Token {
       next(error);
     }
   }
+
+  /**
+   *
+   *
+   * @static
+   * @param {*} req
+   * @param {*} res
+   * @param {*} next
+   * @return {null} Returns nothing
+   */
+  static async authenticate(req, res, next) {
```

Comment:
Why is this here?

Is it any part of your ticket?

Reply:
Yes. 
My ticket requires that I authenticate a user if the user is signed in so that I will be able to log the article into the user's read history

If I used the authorize middleware, it will not allow a logged out user to access the route, so I created the authentication middleware so that it identifies the user is he is logged in, but ignores if the user is not.